big_tests/tests/graphql_metric_SUITE.erl

```diff
@@ -126,6 +140,30 @@ get_by_name_global_erlang_metrics(Config) ->
     %% Other metrics are filtered out
     undef = maps:get(ReadsKey, Map, undef).
 
+get_metrics_by_name_empty_args(Config) ->
+    Result = get_metrics([], Config),
+    ParsedResult = get_ok_value([data, metric, getMetrics], Result),
+    lists:foreach(fun check_metric_by_type/1, ParsedResult),
+    [_|_] = ParsedResult.
+
+get_metrics_by_name_empty_string(Config) ->
+    Result = get_metrics([<<>>], Config),
+    ParsedResult = get_ok_value([data, metric, getMetrics], Result),
+    lists:foreach(fun check_metric_by_type/1, ParsedResult),
```

Comment:
One last thing: no need for foreach, if it is an empty list.

Because it is as useful as code like that (not so useful):
```
lists:foreach(fun check_metric_by_type/1, []),
```